aten/src/ATen/native/native_functions.yaml

```diff
@@ -476,6 +476,11 @@
 
 - func: batch_norm(Tensor input, Tensor? weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float momentum, float eps, bool cudnn_enabled) -> Tensor
 
+- func: quantized_batch_norm(Tensor input, Tensor? weight, Tensor? bias, Tensor mean, Tensor var, float eps, float output_scale, int output_zero_point) -> Tensor
```

Comment:
We should also register the variant with the relu (quantized_batch_norm_relu), like other cases:https://github.com/pytorch/pytorch/blob/ecbf6f99e6a4e373105133b31534c9fb50f2acca/aten/src/ATen/native/quantized/cpu/qconv.cpp#L640

Reply:
Let's leave it at quantized batch_norm support for now and add relu fusion later (if required)